### ISSUE TITLE
[WASM] Font preloading support 

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#693]
## I'm submitting a...
- Feature request

## Current behavior
Fonts specified as WebFonts do not get loaded synchronously, making TextBlock based control measure their content incorrectly. Loading fonts using a data-uri is not a workaround.

## Expected behavior
Fonts are loaded and measured properly.

## Possible workaround

This works as a workaround, forcing fonts to load very early in the app's life cycle:
```css
body::after {
    font-family: 'Symbols';
    background: transparent;
    content: "";
    opacity: 0;
    pointer-events: none;
    position: absolute;
}
```

## Environment

```
Nuget Package: 

Package Version(s): 

Affected platform(s):
- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [ ] 2017 Preview (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )
```

**Comment by zipswich** [comment]
![image](https://user-images.githubusercontent.com/6095671/67943651-d9e52600-fbb0-11e9-804a-8a67fc81abe9.png)

**Comment by carldebilly** [comment]
`FontFace.load()` could probably be used to fix this problem.
Doc: https://developer.mozilla.org/en-US/docs/Web/API/FontFace/load

**Comment by carldebilly** [comment]
The [`FontFaceSet`](https://developer.mozilla.org/en-US/docs/Web/API/FontFaceSet) may also help on this issue.

**Comment by agneszitte-nventive** [comment]
@jeromelaban / @carldebilly Interesting article here in my opinion:
https://nooshu.com/blog/2021/01/23/the-importance-of-font-face-source-order-when-used-with-preload/

**Comment by agneszitte-nventive** [comment]
(@jeromelaban FYI, there is this item also https://github.com/unoplatform/Uno.Themes/issues/733)

**Comment by jeromelaban** [comment]
Available in 4.6
